nautobot/docs/installation/nautobot.md

```diff
@@ -0,0 +1,255 @@
+
+```no-highlight
+(nautobot) $ pip3 install --upgrade pip wheel
+```
+
+## Install Nautobot
+
+Use Pip to install Nautobot:
+
+```no-highlight
+(nautobot) $ pip3 install nautobot
+```
+
+Great! We have a virtualenv ready for use by the `nautobot` user, so let's proceed to verifying the installation.
+
+## Verify your Nautobot Installation
+
+You should now have a fancy `nautobot-server` command in your environment. This will be your gateway to all things Nautobot! Run it to confirm the installed version of `nautobot`:
+
+```no-highlight
+(nautobot) $ nautobot-server --version
+```
+
+## Configuration
+
+Before you can use Nautobot, you'll need to configure it by telling it where your database and Redis servers can be found, among other things. This is done with the `nautobot_config.py` configuration file.
+
+### Initialize your configuration
+
+Initialize a new configuration by running `nautobot-server init`. You may specify an alternate location and detailed instructions for this are covered in the documentation on [Nautobot Configuration](../../configuration).
+
+However, because we've set the `NAUTOBOT_ROOT`, this command will automatically create a new `nautobot_config.py` at the default location based on this at `/opt/nautobot/nautobot_config.py`:
+
+```no-highlight
+(nautobot) $ nautobot-server init
+```
+
+### Required Settings
+
+Your `nautobot_config.py` provides sane defaults for all of the configuration settings. You will inevitably need to update the settings for your environment.
+
+Prepare to edit `/opt/nautobot/nautobot_config.py`, and head over to the documentation on [Required Settings](../../configuration/required-settings) to tweak your required settings.
+
+Save your changes to your `nautobot_config.py` and then return here.
+
+## Optional Settings
+
+All Python packages required by Nautobot will be installed automatically when running `pip3 install nautobot`.
+
+Nautobot also supports the ability to install optional Python packages. If desired, these packages should be listed in `local_requirements.txt` within the `NAUTOBOT_ROOT` directory at `/opt/nautobot/local_requirements.txt`.
+
+If you decide to use any [Nautobot plugins](../../plugins), they should be listed in this file.
+
+We will cover two examples of common optional settings below.
+
+### Configuring NAPALM
+
+Nautobot provides built-in support for the [NAPALM automation](https://napalm-automation.net/) library, which allows Nautobot to fetch live data from devices and return it to a requester via its REST API. The [`NAPALM_USERNAME`](../../configuration/optional-settings#napalm_username) and [`NAPALM_PASSWORD`](../../configuration/optional-settings#napalm_password) configuration parameters define the credentials to be used when connecting to a device.
+
+To use NAPALM, add `napalm` to your `local_requirements.txt` so that it can be installed and kept up to date:
+
+```no-highlight
+(nautobot) $ echo napalm >> /opt/nautobot/local_requirements.txt
+```
```

Comment:
add a `pip install -r /opt/nautobot/local_requirements.txt`

Reply:
That's covered just after this section.